redis_websocket_api/server.py

```diff
@@ -78,11 +78,14 @@ def __init__(self, redis, subscriber, read_timeout=None, keep_alive_timeout=None
         """Pass messages from subscribed channels to handlers."""
 
         async for channel, msg in self.receiver.iter(encoding="utf-8"):
+            if channel.is_pattern:
+                channel_name, msg = msg[0].decode(), msg[1]
```

Comment:
I assume this is correct because redis will send back the pattern name to which I subscribed,  as well as the channel that the message was actually published on (and the message itself)?
So `channel` holds the pattern (e.g. `"weather:*"`) and msg holds a list (or any other indexable data structure) with the contents `["weather:Berlin", "2°C"]`? 
That's what I could gather from [here](https://redislabs.com/redis-best-practices/communication-patterns/pub-sub/).
Please confirm my suspicion, then I will merge. If I am wrong however, please explain how you came to choose these indexes.

Reply:
Yes, that's exactly what happens: `channel.name` is the pattern for `PSUBSCRIBE` so aioredis returns a 2-tuple with the senders channel and the message content as payload (`msg`).

Reply:
ok, thank you. Looks good to me then.